nextflow.config.example

```diff
@@ -1,17 +1,23 @@
+// Please Copy This File to `nextflow.config`
+// And Complete Placeholder Marked With <BRACKET>,
+// Examples Are In Comment
+// NOTE: It's intended that nextflow.config file is not tracked by git.
+
 params {
     run_id = 1 
 
     ref_ver = "hg19"
     ref_assembly =  params.ref_ver == "hg19" ? "grch37" : "grch38"
-    ref_dir = "/home/hwan/aim_data_dependencies_04_24_2024_sasi"
 
-    input_vcf = "/home/hwan/AI_MARRVEL/test_data/demo.vcf.gz"
-    input_hpo = "/home/hwan/AI_MARRVEL/test_data/hpos.txt"
 
-    outdir = "/home/hwan/AI_MARRVEL/test_output_060624_2/"
+    ref_dir = <PATH_TO_REFERENCE_DIRECTORY> // Example: "/home/jaeyeon/Data/aim_data_dependencies/"
```

Comment:
Can we put environment value instead of?

Reply:
I will check if it's possible

Reply:
Yes it can, but I think it's better to use parameter rather than environment variables.

```
nextflow run . --refdir /home/jaeyeon/Data/aim_data_dependencies/
```
And if we use parameters, then there's no need to untrack config files.

How do you think about this?

Reply:
Then put empty strings instead of the placeholders.

Reply:
Had discission and here are things to do:
- [ ] keep `nextflow.config`
- [ ] create `params.json.example`, and instruct people to copy it, and how to use it
- [ ] Edit `README.md`